lib/index.js

```diff
@@ -22,6 +22,9 @@ function register (server, options, next) {
         if (err) {
           return cb(err, null);
         }
+        if (rows.length === 0) { // i.e. no orgs -OR- filter --> wrong tagId or no orgs data
+          return cb(err, null);
```

Comment:
Null is probably fine, but I was wondering if an error would be suitable? I am not sure though, would be keen to hear your and Simons thoughts.